wordpress-mobile/WordPress-iOS
# Adds animation to the Jetpack Powered banner

Fixes #NA

This PR adds animation to the Jetpack Powered banner. The banner will hide when scrolling down, and show when scrolling back up.

**NOTE: this PR only adds the animation on Stats. The notifications banner will be addressed in a future PR**

<p align=center>

<img height=500 src=https://user-images.githubusercontent.com/34376330/178846907-68209ce9-dd27-4489-a5b8-f49fabccc46f.gif>
</p>
To test:

- Build/run this branch
- Go to My Site -> Stats
- Select any tab (Insights, Days, etc)
- Make sure the "Jetpack powered" banner hides when scrolling down, and shows when scrolling up (see animation above)

## Regression Notes
1. Potential unintended areas of impact
Low, this just adds animation to the Jetpack Powered banner

2. What I did to test those areas of impact (or what existing automated tests I relied on)
test on a variety of devices

3. What automated tests I added (or what prevented me from doing so)
none
PR submission checklist:
